### DISCUSSION TITLE
getBounds on map move

### DISCUSSION THREAD

**Discussion by cropcrop** [uber/react-map-gl#1521] [General]
I'm using V3, is there any way to get mapbox bounds while moving the map?

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/docs/api-reference/web-mercator-viewport#getboundsoptions

**Reply by cropcrop** [reply]
Thank you @Pessimistress!

**Reply by bitfella** [reply]
> https://visgl.github.io/react-map-gl/docs/api-reference/web-mercator-viewport#getboundsoptions

This link isn't working anymore unfortunately. Could you please provide an updated one? Thanks @Pessimistress

**Reply by spencerhcheng** [reply]
Can you please provide an updating link? @Pessimistress TIA
